knowledge/studios/bbc_studios/qna.yaml

```diff
@@ -0,0 +1,25 @@
+created_by: 
+domain: television
+seed_examples:
+- answer: | BBC Studios Productions brings together the majority of BBC Television's former in-house production departments; Factual, Drama, Comedy (both combined as Scripted in the new division), Entertainment, and Music & Events.[4] BBC Children's production is set to move into BBC Studios Productions from April 2022 to increase the potential of taking British children's content to the wider global market, along with BBC Three's in-house production team, which is joining from April 2021.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!